mne/io/base.py

```diff
@@ -658,7 +658,7 @@ def filenames(self):
         return tuple(self._filenames)
 
     @annotations.setter
-    def annotations(self, annotations):
+    def annotations(self, annotations, emit_warning=True):
```

Comment:
There is no user-exposed param, it's just for internal calls like `raw.crop()` that should never emit such warnings (i.e., of course if you crop you will lose data + annotations)